events/models.py

```diff
@@ -241,6 +243,13 @@ def finish_date(self):
         else:
             return self.end_date
 
+    @property
+    def to_be_resolved(self):
+        """
+        Return True if event is waiting to be resolved.
+        """
+        return datetime.now(pytz.timezone(settings.TIME_ZONE)) >= self.finish_date
```

Comment:
If finish_date is timezone aware, I guess there's no need for explicitly using timezone from settings, simply `django.utils.timezone.now` will return timezone aware datetime object, that can be compared to finish_date